lib/web_ui/dev/licenses.dart

```diff
@@ -79,13 +79,18 @@ class LicensesCommand extends Command<bool> {
   }
 
   List<io.File> _flatListSourceFiles(io.Directory directory) {
+    // This is the old path that tests used to be built into. Ignore anything
+    // within this path.
+    final String legacyBuildPath = path.join(environment.webUiRootDir.path, 'build');
```

Comment:
Should we also add the legacy path to the `felt clean` command?
https://github.com/flutter/engine/blob/b2720b7e9887ff0c000d43f5323c9b4995df7831/lib/web_ui/dev/clean.dart#L40-L49

I personally rarely use `felt clean` so I don't have a strong opinion here.

Reply:
Yeah good idea